packages/nuxt/src/components/loader.ts

```diff
@@ -15,6 +15,7 @@ interface LoaderOptions {
   sourcemap?: boolean
   transform?: ComponentsOptions['transform']
   experimentalComponentIslands?: boolean
+  isDev: boolean
```

Comment:
@danielroe We could probably also do it with nuxt.options.test ?

Reply:
makes sense! 👍